src/seedu/addressbook/commands/Command.java

```diff
@@ -14,6 +14,7 @@
 public class Command {
     protected AddressBook addressBook;
     protected List<? extends ReadOnlyPerson> relevantPersons;
+    protected boolean autoListEnabled = false;
```

Comment:
In this case, it may make sense to select a default; but, in general, you need to be careful in selecting the default behavior, particularly if it is chosen in the parent class.

The best way to handle this is to let the child class define what should be its behavior by default.